### DISCUSSION TITLE
"hideLogs" when inspecting with Safari

### DISCUSSION THREAD

**Discussion by christianhengst** [ionic-team/capacitor#3254] [Q&A]
I'm looking for a way to hide **all** Capacitor logs when inspecting my app remotely with Safari. 

Using the `"hideLogs": true` option hides logs successfully in Xcode but not in Safari. 
My console is filled with `▶︎ native` and `▶︎ result` logs, which can be quite confusing.

Is there a way to hide these logs?

**Comment by aparajita** [comment]
Currently it does not appear to be possible. You should file an issue about this.

**Comment by jcesarmobile** [comment]
It's not possible at the moment, it has been requested on https://github.com/ionic-team/capacitor/issues/3723
